src/NuGetForUnity/Editor/NugetPreferences.cs

```diff
@@ -72,6 +74,34 @@ public override void OnGUI(string searchContext)
                 NugetHelper.NugetConfigFile.Verbose = verbose;
             }
 
+            EditorGUILayout.BeginHorizontal();
+            {
+                var packagesConfigPath = NugetHelper.NugetConfigFile.PackagesConfigDirectoryPath;
+                EditorGUILayout.LabelField(
+                    new GUIContent($"Packages Config path: {NugetHelper.NugetConfigFile.RelativePackagesConfigPath}", $"Absolute path: {packagesConfigPath}"));
+                if (GUILayout.Button("Browse"))
+                {
+                    var newPath = EditorUtility.OpenFolderPanel("Select Folder", packagesConfigPath, "");
+
+                    if (!string.IsNullOrEmpty(newPath) && newPath != packagesConfigPath)
+                    {
+                        var pathCheck = NugetHelper.GetAssetsRelativePath(newPath);
+
+                        // if the path root is different or it is not under Assets folder, we want to show a warning message
+                        shouldShowPackagesConfigPathWarning = pathCheck == newPath || pathCheck.StartsWith("..");
+
+                        PackagesConfigFile.Move(newPath);
+                        preferencesChangedThisFrame = true;
+                    }
+                }
+            }
+            EditorGUILayout.EndHorizontal();
+            if (shouldShowPackagesConfigPathWarning)
+            {
+                EditorGUILayout.HelpBox(
+                    "Putting packages.config outside of Assets folder disables the functionality of restoring packages if the file is changed on the disk.", MessageType.Warning);
```

Comment:
```suggestion
                    "The packages.config is placed outside the Assets folder, this disables the functionality of automatically restoring packages if the file is changed on the disk.", MessageType.Warning);
```
Something like this?